### ISSUE TITLE
Local Development Doc Update 

### ISSUE THREAD

**Issue by philoSurfer** [pantheon-systems/documentation#6134]
Re: [Local Development](https://pantheon.io/docs/local-development)

Priority: Medium

## Issue Description:
Missing Drupal 8.x,9.x recommendations for local settings configuration files 
## Suggested Resolution
I don't know if this is now a `settings.local.yml` thing or if it is still a `settings.local.php` and the Drupal 7 example works just fine, but we are only calling out Drupal 7 on this page, might just need to make a note here saying this is supporting for D7-9?

**Comment by alexfornuto** [comment]
We'll clarify here that the local settings file is created automatically for other CMS's, and need only be manually defined for D7.

**Comment by EdwardAngert** [comment]
thanks, @philoSurfer! 
in starting to test out your suggestion, I found that there are a few things I should update about it (including your suggestion)

in particular, `terminus backup:get` gets a response with a link now (not sure what old behavior was), and when I `wget` it, it doesn't do what I want, so I'm still trying to put together some clean bash commands for that section:

```bash
❯ terminus backup:get $site.dev --element=db
https://storage.googleapis.com/gcs-pantheon-backups/...
```

so, I'm claiming this issue for now, and I'll try to get some work into it

**Comment by EdwardAngert** [comment]
updating to say we should ignore my previous comment - after more testing, I realized where I messed up
the original issue was closed in #6517 
incoming PR will close this issue, but is more cosmetic than anything else
